src/unittest/test_serialization.cpp

```diff
@@ -660,7 +661,7 @@ void TestSerialization::testFloatFormat()
 		}
 
 		fs = u32Tof32Slow(v.second);
-		if (std::abs(v.first - fs) > std::abs(v.first * 0.000005f)) {
+		if (std__fabs(v.first - fs) > std::fabs(v.first * 0.000005f)) {
```

Comment:
std__fabs?

Reply:
C++28 feature /s

Such stuff happens when you 1) don't test the code and/or 2) are a little bit too tired. Fixed it.